### ISSUE TITLE
Typescript of interest?

### ISSUE THREAD

**Issue by 4imble** [gothinkster/aurelia-realworld-example-app#25]
I forked the project, created a new Aurelia application within using TypeScript and then copied over and converted all the JS files to TypeScript a couple of weeks ago. I did not go as far as setting all the Types, but only did the ones that were obvious or needed to be fixed up as part of the conversion.

I will ultimately be using my fork as a foundation to build my own real blog but wondered if it would be of use.

**Comment by loaded02** [comment]
Yes of course. Sounds good. Since we cant merge your fork, what would you think if we have a link to your fork in the readme?

**Comment by loaded02** [comment]
@4imble do you want to create a pr to a [typescript branch](https://github.com/gothinkster/aurelia-realworld-example-app/tree/typescript)?

**Comment by 4imble** [comment]
Maybe, I have contacted you in gitter if you'd like to chat about it :)

**Comment by 4imble** [comment]
I'm happy to merge in my fork and I'll try and keep it up to date as changes go into master. Thinking about it, I suspect there wont be too many as only bug fixes and optimisations should only be going in, not new features.

I'll try and get the changes that have happened so far in and do the pull request this week.

**Comment by 4imble** [comment]
As i mentioned in the pull request you have waiting, do you want me to also convert to webpack?

**Comment by loaded02** [comment]
Hi @4imble! Sorry for the delayed reply. It looks like I will do that. Size is smaller. I wanted to do a speed comparisson before I merge. But I am pretty sure we gonna change to wp. Thanks for your work.

**Comment by 4imble** [comment]
No need to apologise, we're all busy :)
I stay after work to do personal projects on Tuesday, I'll dedicate some time to updating to WP and bringing in the new changes since my fork. Cheers 👍

**Comment by 4imble** [comment]
Enough has changed that it's probably easier for me to do the conversion on each file from scratch again. 
Also, I'm not sure how best to keep the two branches updated without manually applying each commit. I've been pretty slow, but is it worth holding off even longer and then doing the conversion when we think it's all done 99% and unlikely to change?

**Comment by loaded02** [comment]
It would be best to commit your status quo to the typescript branch. I have no good way to keep 3 branches in sync (webpack, amd, typescript). I‘d say we have a master and the other branches as spin offs. Is that good enough?

**Comment by 4imble** [comment]
Looks like the changes are quite settled now. Do you want to make me a contributor to the typescript branch? I'll have a go at updating it again with the latest changes. Cheers 👍

**Comment by loaded02** [comment]
@4imble please feel free to make a pr to the typescript branch. This way you‘ll become a contributor. Thanks for your work!
